pwiz_tools/Skyline/TestData/CommandLineTest.cs

```diff
@@ -664,6 +664,69 @@ public void ConsoleExportSpecLibTest()
 
         }
 
+        [TestMethod]

```

Comment:
Be sure to use SkylineTester to run this new test in all 5 languages before merging this PR.

Reply:
Thanks, I addressed this by adding new international testing files and using GetTestPathLocale()